pod/perldata.pod

```diff
@@ -606,7 +606,11 @@ X<line> X<file> X<package>
 
 The two control characters ^D and ^Z, and the tokens __END__ and __DATA__
 may be used to indicate the logical end of the script before the actual
-end of file.  Any following text is ignored.
+end of file.  Any following text is ignored, except with __DATA__:
+
+   while (<DATA>) { print; }
```

Comment:
We shouldn't add examples using while-readline with `$_` since it does not localize that variable, instead suggest:

    while (my $line = <DATA>) { print $line; }

Reply:
I like $line, but I was able to get away with $_ just fine, so users
should be told when they can still use $_, and when they shouldn't.
Else everybody will just use trial and error.

Also regarding "Any following text is ignored."
Well my example proves that it is not.
As far as we beginners are concerned.
So if indeed your want to say it is ignored,
then you need to explain more, else it seems
the article was written at 5:00 AM.

Reply:
The explanation isn't relevant to this section since it's not a section about using while-readline. We should simply use examples that don't promote bad practices.

The text is always ignored by the interpreter, the original text is correct, though it could probably use more explanation. In other words, text following `__DATA__` or `__END__` is never parsed and executed as part of the perl program.

Reply:
Sounds good. Please make the edit. By the way, I don't recall seeing a minimal __END__ example anywhere either...